blocks/share-bar-block/features/share-bar/default.jsx

```diff
@@ -97,7 +100,7 @@ const ShareBarContainer = () => {
 };
 
 export const ShareBar = ({
-  customFields, websiteName, websiteDomain, websiteUrl, headlineString,
+  customFields = {}, websiteName, websiteDomain, websiteUrl, headlineString,
```

Comment:
is this new?

Reply:
Yes, a small update to fix and existing issue where the share bar renders with a blank div with box shadow

Reply:
Also updated as it's a small component which allows me to use as reference, but issue was it had this issue I needed to fix